client/my-sites/plugins/main.jsx

```diff
@@ -383,12 +384,10 @@ export class PluginsMain extends Component {
 		return (
 			<Button
 				className="plugins__button"
-				compact
 				href={ browserUrl }
 				onClick={ this.handleAddPluginButtonClick }
 			>
-				<Gridicon icon="plus" />
-				<span className="plugins__button-text">{ translate( 'Add Plugin' ) }</span>
+				<span className="plugins__button-text">{ translate( 'Browse plugins' ) }</span>
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **28** times:
`translate( 'Browse all plugins', { context: 'button label'} )` *ES Score: 6*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/plugin-button-placement)